lib/screens/order_detail_page.dart

```diff
@@ -91,7 +91,7 @@ class _OrderDetailPageState extends State<OrderDetailPage> {
             ),
           ),
           _buildOrderItemList(order.items),
-          _buildOrderPriceDescriptionBar(order.getOrderPrice()),
+          _buildOrderPriceDescriptionBar(order.totalPrice),
```

Comment:
@dragonfly91 no need to rename this method to `_buildTotalPriceDescriptionBar`? :)

Reply:
Well, since total price = order price, no need I guess haha! 😂

Reply:
Ok~👍 (I was thinking about consistency in naming but probably it's not a big deal😝) Will approve the PR!